### DISCUSSION TITLE
How to override the default line-height 1.5rem to root element

### DISCUSSION THREAD

**Discussion by Aadhisivam** [tailwindlabs/tailwindcss#10070] [Help] [answered]
Getting default 1.5rem at root element, I wish to override this value.. can someone help

**Comment by wongjn** [comment, marked as answer]
You should be able to override it by adding a [`leading-` utility class](https://tailwindcss.com/docs/line-height) on the element.

**Reply by Aadhisivam** [reply]
Is there anyway to change that default through `tailwind.config.js`

**Reply by wongjn** [reply]
Yes. You could [disable the core preflight plugin](https://tailwindcss.com/docs/preflight#disabling-preflight) and then reimplement it but remove the `line-height` declaration before passing the content to `addBase()`.

**Comment by adamwathan** [comment]
I would just add some CSS:

```
@layer base {
  html {
    line-height: 1.25;
  }
}
```
